src/common/utils/getLocations.js

```diff
@@ -0,0 +1,11 @@
+import {
+  batchFetch,
+} from '@folio/stripes-acq-components';
+
+export function getLocations(mutatorLocations, pieces, poLine) {
```

Comment:
it's more like `fetchLocations`, `get` doesn't tell you exactly from where are you going to get these locations